1-js/01-getting-started/3-code-editors/article.md

```diff
@@ -12,12 +12,12 @@ An IDE loads the project (which can be many files), allows navigation between fi
 
 If you haven't selected an IDE yet, consider the following options:
 
-- [Visual Studio Code](https://code.visualstudio.com/) (cross-platform, free).
+- [Visual Studio](https://visualstudio.microsoft.com) (cross-platform, free/paid).
 - [WebStorm](http://www.jetbrains.com/webstorm/) (cross-platform, paid).
 
-For Windows, there's also "Visual Studio", not to be confused with "Visual Studio Code". "Visual Studio" is a paid and mighty Windows-only editor, well-suited for the .NET platform. It's also good at JavaScript. There's also a free version [Visual Studio Community](https://www.visualstudio.com/vs/community/).
+There's also "Visual Studio Code", not to be confused with "Visual Studio". "Visual Studio" is paid and well-suited IDE for the .NET platform. It's also good at JavaScript. Detailed information about their difference can be found in [Visual Studio Code FAQ](https://code.visualstudio.com/docs/supporting/faq) page.
```

Comment:
```suggestion
There's also "Visual Studio Code", not to be confused with "Visual Studio". "Visual Studio" is paid and well-suited IDE for the .NET platform. It's also good for JavaScript. There is also a free edition of "Visual Studio" called [Visual Studio Community](https://www.visualstudio.com/vs/community/), albeit with limited features. Detailed information about the difference between "Visual Studio" and "Visual Studio Code" can be found in [Visual Studio Code FAQ](https://code.visualstudio.com/docs/supporting/faq) page.
```

I think we should mention about free version as many tend to skip softwares when they see the word "paid".

Is it "at JavaScript" or "for JavaScript"?

Reply:
Btw, feel free to restructure the paragraph as you see fit! :-)

Reply:
Hi @MuhammedZakir,

Thanks for the feedback. I totally agree with all of them.

Talking about Visual Studio & VSCode as a paragraph under code editors part seems unrelated for me too. Therefore, It may be better to remove those sentences as you mentioned.